### PR TITLE
Enabling or disabling the standard HTML5 select

### PR DESCRIPTION
**Pull request by saade**
- [x] Changes have been thoroughly tested to not break existing functionality.
- [x] New functionality has been documented or existing documentation has been updated to reflect changes.
- [x] Visual changes are explained in the PR description using a screenshot/recording of before and after.

This PR adds a `->native()` option to select fields. This is nice because we can disable the default HTML5 select and keep a visual consistency across the entire project.

https://github.com/filamentphp/filament/assets/14329460/c6e733af-f907-48b6-b193-dd7cf5e5aa16

### CODE REVIEW

**Comment by saade** [comment]
@danharrin didn't have the time to look further into your `AutoScreenshot` feature, so I didn't add it to the docs :) I can PR this later

File: packages/forms/resources/js/components/select.js [hunk 1]
```diff
@@ -39,6 +40,8 @@ export default function selectFormComponent({
         state,
 
         init: async function () {
+            console.log(isSearchable)
```
Comment (zepfietje) [inline packages/forms/resources/js/components/select.js:43]: Make sure to remove this next time. 😉 I've done that now.

Comment (zepfietje) [reply packages/forms/resources/js/components/select.js:43]: Btw, really like this small PR. 😄

Comment (danharrin) [reply packages/forms/resources/js/components/select.js:43]: LOL im sorry

Comment (danharrin) [reply packages/forms/resources/js/components/select.js:43]: that was me

**Pull request merged**